test/suites/dev-tanssi-relay/xcm/test-export-message-from-para.ts

```diff
@@ -190,5 +190,175 @@ describeSuite({
+                    parents: 1,
+                    interior: {
+                        X3: [
+                            {
+                                GlobalConsensus: {
+                                    ByGenesis: "0x983a1a72503d6cc3636776747ec627172b51272bf45e50a355348facb67a820a",
+                                },
+                            },
+                            {
+                                Parachain: 2005, // Incorrect parachain ID
+                            },
+                            {
+                                PalletInstance: 5,
+                            },
+                        ],
+                    },
+                };
+                const xcmToExport = new XcmFragment({
+                    assets: [
+                        {
+                            multilocation: incorrectContainerAsset,
+                            fungible: transferredBalance / 10n,
+                        },
+                    ],
+                    beneficiary: "0x983a1a72503d6cc3636776747ec627172b51272b",
+                })
+                    .reserve_asset_deposited()
+                    .clear_origin()
+                    .buy_execution()
+                    .deposit_asset_v3()
+                    .set_topic();
+
+                const xcmMessage = new XcmFragment({
+                    assets: [
+                        {
+                            multilocation: {
+                                parents: 0,
+                                interior: { Here: null },
+                            },
+                            fungible: transferredBalance / 10n,
+                        },
+                    ],
+                    beneficiary: u8aToHex(random.addressRaw),
+                })
+                    .withdraw_asset()
+                    .buy_execution()
+                    .export_message(xcmToExport.instructions, ethereumNetwork, "Here")
+                    .as_v3();
+
+                await context.createBlock();
+                // Send RPC call to enable para inherent candidate generation
+
+                await customDevRpcRequest("mock_enableParaInherentCandidate", []);
+
+                // Send ump message
+                await injectUmpMessageAndSeal(context, {
+                    type: "XcmVersionedXcm",
+                    payload: xcmMessage,
+                } as RawXcmMessage);
+
+                const tokenTransferNonceBefore =
+                    await polkadotJs.query.ethereumOutboundQueue.nonce(tokenTransferChannel);
+
+                // Wait until message is processed
+                await jumpToSession(context, 3);
+                await context.createBlock();
+                const tokenTransferNonceAfter =
+                    await polkadotJs.query.ethereumOutboundQueue.nonce(tokenTransferChannel);
+
+                expect(tokenTransferNonceAfter.toBigInt()).toBe(tokenTransferNonceBefore.toBigInt()); // No change in nonce
```

Comment:
Maybe we can also find the emitted error, if any?

Reply:
https://github.com/moondance-labs/tanssi/pull/1243/commits/15bcea9ea96c8e93440bbf5ab72d2058fc44dae5